docs/quickstart/custom_dataset.md

```diff
@@ -1,31 +1,46 @@
 # Using custom data
 
-Training model on existing datasets is only so fun. If you would like to train on self captured data you will need to process the data into an existing format. Specifically we need to know the camera poses for each image. [COLMAP](https://github.com/colmap/colmap) is a standard tool for extracting poses. It is possible to use other methods like [SLAM](https://en.wikipedia.org/wiki/Simultaneous_localization_and_mapping) or hardware recorded poses. We intend to add documentation for these other methods in the future.
+Training model on existing datasets is only so fun. If you would like to train on self captured data you will need to process the data into the nerfstudio format. Specifically we need to know the camera poses for each image.
 
-## Nerfstudio dataset
-
-To assist running on custom data we have a script that will process a video or folder of images into a format that is compatible with nerfstudio. We use [COLMAP](https://colmap.github.io) and [FFmpeg](https://ffmpeg.org/download.html) in our data processing script, please have these installed. We have provided a quickstart to installing COLMAP below, FFmpeg can be downloaded from [here](https://ffmpeg.org/download.html)
-
-To process your data run:
+To process your own data run:
 
 ```bash
-ns-process-data {video,images,insta360,record3d} --data {DATA_PATH} --output-dir {PROCESSED_DATA_DIR}
+ns-process-data {video,images,polycam,insta360,record3d} --data {DATA_PATH} --output-dir {PROCESSED_DATA_DIR}
 ```
 
 A full set of arguments can be found {doc}`here</reference/cli/ns_process_data>`.
 
+We Currently support the following custom data types:
+| Data | Requirements | Preprocessing Speed |
+| -------- | ----------------------- | ------------------- |
+| [Images](images_and_video) | COLMAP | 🐢 |
+| [Video](images_and_video) | COLMAP | 🐢 |
+| [Polycam](polycam) | LiDAR iOS Device | 🐇 |
+| [Record3D](record3d) | LiDAR iOS Device | 🐇 |
+| Insta360 | 2 Sensor camera, COLMAP | 🐢 |
+
+(images_and_video)=
```

Comment:
delete

Reply:
This might be a RST thing?

Reply:
It is needed for referencing (lines 16,17)

Reply:
It is needed for referencing.